socorro/cron/crontabber_app.py

```diff
@@ -720,18 +689,6 @@ def main(self):
                 return 3
         return 0
 
-    @staticmethod
-    def _reorder_class_list(class_list):
-        # class_list looks something like this:
-        # [('FooBarJob', <class 'FooBarJob'>),
-        #  ('BarJob', <class 'BarJob'>),
-        #  ('FooJob', <class 'FooJob'>)]
-        return reorder_dag(
-            class_list,
-            depends_getter=lambda x: getattr(x[1], 'depends_on', None),
-            name_getter=lambda x: x[1].app_name
-        )
```

Comment:
I removed the rest of the `depends_on` related code, so we don't need this anymore.